addons/html_builder/static/src/plugins/header_navbar_option.xml

```diff
@@ -0,0 +1,161 @@
+             <BuilderColorPicker action="'customizeWebsiteVariable'" actionParam="'header-text-color'"/>
+             <BuilderSelect applyTo="'#top_menu'" action="'websiteConfig'">
+                 <BuilderSelectItem 
+                     id="'header_alignment_opt'"
+                     actionParam="{ 
+                         views: [], 
+                         vars: {'hamburger-position': 'left'} 
+                     }"
+                     className="'o_we_icon_select'"
+                 >
+                     <i class="fa fa-align-left fa-fw"></i>
+                 </BuilderSelectItem>
+                 <BuilderSelectItem 
+                     t-if="this.currentKey != 'website.template_header_hamburger'"
+                     id="'header_alignment_opt'"
+                     actionParam="{ 
+                         views: ['website.template_header_default_align_center', 'website.template_header_boxed_align_center', 'website.template_header_stretch_align_center', 'website.template_header_search_align_center', 'website.template_header_sales_one_align_center', 'website.template_header_sales_two_align_center', 'website.template_header_sales_four_align_center', 'website.template_header_sidebar_align_center'], 
+                         vars: {'hamburger-position': 'center'} 
+                     }"
+                     className="'o_we_icon_select'"
+                 >
+                     <i class="fa fa-align-center fa-fw"></i>
+                 </BuilderSelectItem>
+                 <BuilderSelectItem 
+                     id="'header_alignment_opt'"
+                     actionParam="{ 
+                         views: ['website.template_header_default_align_right', 'website.template_header_hamburger_align_right', 'website.template_header_boxed_align_right', 'website.template_header_stretch_align_right', 'website.template_header_search_align_right', 'website.template_header_sales_one_align_right', 'website.template_header_sales_two_align_right', 'website.template_header_sales_four_align_right', 'website.template_header_sidebar_align_right'], 
+                         vars: {'hamburger-position': 'right'} 
+                     }"
+                     className="'o_we_icon_select'"
+                 >
+                     <i class="fa fa-align-right fa-fw"></i>
+                 </BuilderSelectItem>
+             </BuilderSelect>
+         </BuilderButtonGroup>
+     </BuilderRow>
+     <BuilderRow label.translate="Link Style" t-if="this.currentKey == 'website.template_header_default' or this.currentKey == 'website.template_header_boxed' or this.currentKey == 'website.template_header_vertical' or this.currentKey == 'website.template_header_search' or this.currentKey == 'website.template_header_sales_one' or this.currentKey == 'website.template_header_sales_two' or this.currentKey == 'website.template_header_sales_three' or this.currentKey == 'website.template_header_sales_four'">
```

Comment:
maybe add an utils to make it less verbous.
Like hasViews(["template_header_default", "template_header_boxed", ....)